bin/brew

```diff
@@ -9,20 +9,43 @@ BREW_FILE_DIRECTORY="$(quiet_cd "${0%/*}/" && pwd -P)"
 HOMEBREW_BREW_FILE="${BREW_FILE_DIRECTORY%/}/${0##*/}"
 
 HOMEBREW_PREFIX="${HOMEBREW_BREW_FILE%/*/*}"
-[[ -n "$HOMEBREW_PREFIX" && "$HOMEBREW_PREFIX" != "$HOMEBREW_BREW_FILE" ]] \
-  || HOMEBREW_PREFIX="/"
+if [[ -z "$HOMEBREW_PREFIX" || "$HOMEBREW_PREFIX" == "$HOMEBREW_BREW_FILE" ]]
```

Comment:
Please use a single `=` for string comparison in shell code (the style we use everywhere else).